geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DestroyRegionCommandTest.java

```diff
@@ -73,4 +83,35 @@ public void whenNoRegionIsFoundOnAnyMembers() throws Exception {
     result = parser.executeCommandWithInstance(command, "destroy region --name=test --if-exists");
     assertThat(result.getStatus()).isEqualTo(Result.Status.OK);
   }
+
+  @Test
+  public void multipleResultReturnedWithOneError() throws Exception {
+    // mock this to pass the member search call
+    doReturn(Collections.singleton(DistributedMember.class)).when(command)
+        .findMembersForRegion(any(), any());
+
+    ResultCollector collector = mock(ResultCollector.class);
+    doReturn(collector).when(command).executeFunction(any(), any(), any(Set.class));
+
+    List<CliFunctionResult> functionResults = new ArrayList<>();
+    doReturn(functionResults).when(collector).getResult();
+    CliFunctionResult result1 = mock(CliFunctionResult.class);
+    CliFunctionResult result2 = mock(CliFunctionResult.class);
+    functionResults.add(result1);
+    functionResults.add(result2);
+
+    when(result1.isSuccessful()).thenReturn(true);
+    when(result1.getMessage()).thenReturn("result1 message");
+    when(result1.getXmlEntity()).thenReturn(mock(XmlEntity.class));
+
+    when(result2.isSuccessful()).thenReturn(false);
+    when(result2.getMessage()).thenReturn("result2 message");
+
+    result = parser.executeCommandWithInstance(command, "destroy region --name=test");
+    assertThat(result.getStatus()).isEqualTo(Result.Status.ERROR);
+    assertThat(result.getContent().toString()).contains("result2 message");
+
+    // verify that xmlEntiry returned by the result1 is not saved to Cluster config
+    verify(command, never()).persistClusterConfiguration(any(), any());
+  }
```

Comment:
Could have a couple more tests, this one multiple-results tests looks like it misses coverage of some conditional clauses in the foreach in `DestroyRegionCommand.destroyRegion`: 1) a result contains a throwable 2) a not-successfull result has a null message.